src/screens/MealList/mealSteps/styles.tsx

```diff
@@ -0,0 +1,65 @@
+import { StyleSheet } from 'react-native';
+
+const styles: any = StyleSheet.create({
+
+    view: {
+        width: '100%',
+        marginLeft: 0,
+        marginRight: 0,
+    },
+    mealImageContainer: {
+        marginTop: 0,
+        alignSelf: 'center',
+        width: '100%',
+        height: 220,
+    },
+    mealImage: {
+        height: 220,
+        width: '100%',
+        flex: 1,
+    },
+    recipeNameCard: {
+        position: 'absolute', 
```

Comment:
I think this is going to give us issues on different screen sizes. You should almost never need to use absolute positioning